src/__tests__/resources/assets.integration.spec.ts

```diff
@@ -256,4 +256,10 @@ describe('Asset integration test', () => {
       .autoPagingToArray({ limit });
     expect(result.length).toBe(limit);
   });
+
+  test('list partition', async () => {
+    const limit = 5;
+    const result = await client.assets.list({ partition: '1/10', limit });
```

Comment:
does it work well with autoPaging methods?

Reply:
I dont think so. Partition is only used when you dont want pagination (fetch everything ignoring limits). So i dont think we should think about that case?

Reply:
Partition should work excellent with pagination?
Essentially you just divide the data set in the backend before fetching so still need pagination for each partition